pkg/asset/installconfig/powervs/session.go

```diff
@@ -42,35 +48,67 @@ type Session struct {
      4) put it into Platform {userid: , iamtoken: , ...}
 */
 func GetSession() (*Session, error) {
-	s, err := getPISession()
+	s, uc, err := getPISession()
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to load credentials")
 	}
-
-	return &Session{Session: s}, nil
+	return &Session{Session: s, Creds: uc}, nil
 }
 
 /*
 //  https://github.com/IBM-Cloud/power-go-client/blob/master/ibmpisession/ibmpowersession.go
 */
-func getPISession() (*ibmpisession.IBMPISession, error) {
+func getPISession() (*ibmpisession.IBMPISession, *UserCredentials, error) {
 
 	var (
-		id, passwd, region, zone string
+		id, passwd, apikey, region, zone string
 	)
 
 	if id = os.Getenv("IBMID"); len(id) == 0 {
-		return nil, errors.New("empty IBMID environment variable")
+		err := survey.Ask([]*survey.Question{
+			{
+				Prompt: &survey.Input{
+					Message: "IBM Cloud User ID",
+					Help:    "The login for \nhttps://cloud.ibm.com/",
+				},
+			},
+		}, &id)
+		if err != nil {
+			return nil, nil, errors.New("empty IBMID environment variable")
+		}
 	}
 	if passwd = os.Getenv("IBMID_PASSWORD"); len(passwd) == 0 {
-		return nil, errors.New("empty IBMID_PASSWORD variable")
-	}
-
+		err := survey.Ask([]*survey.Question{
+                        {
+                                Prompt: &survey.Password{
+                                        Message: "IBM Cloud User Password",
+                                        Help:    "The login for \nhttps://cloud.ibm.com/",
+                                },
+                        },
+                }, &passwd)
+                if err != nil {
+                        return nil, nil, errors.New("empty IBMID_PASSWORD environment variable")
+                }
+        }
+        if apikey = os.Getenv("API_KEY"); len(apikey) == 0 {
```

Comment:
glad to see you found this var. can we just delete the IBMID_PASSWORD logic altogether, and use this only? i didn't know when i did the initial commit that this existed

Reply:
Sorry, I dont understand this comment.
1. We needed the api key, so I look for one defined, if not I prompt for it.  
2. I can change the variable name to whatever you want
3. What are you looking at for your initial commit, we seem to have multiple variables for the same information, Id like to know what you are standardizing to
4. I have not looked into where that password goes, and was told that api key could replace user id I am uncertain if it would also replace password.  We seem to need to carry the user id tho
5. We .ay want to make this more sophisticated then just removing the password.  It may be beneficial to use the password in lieu of an api key

Reply:
i mean, we have been using the password var to store the apikey, but that is misleading. we never use a password for auth. so i'd like to remove the use of the password env var completely. for what we'll standardize to, we can discuss. there are other moving parts that we should coordinate that list with. but for sure, i don't want to continue to use the password var in the installer